app/api/srch/typeahead.rb

```diff
@@ -0,0 +1,124 @@
+require 'grape'
+require 'grape-entity'
+
+module Srch
+  class Typeahead < Grape::API
+
+    # Endpoint definitions
+    resource :typeahead do
+
+      # Request URL should be /api/typeahead/all?srchString=QRY&seq=KEYCOUNT
+      # Basic implementation from classic plots2 SearchController
```

Comment:
I'm not sure what the convention is here, but could we show or link to an example output, so it's a little easier for people to design their API calls against it?